setup.py

```diff
@@ -113,6 +113,7 @@ def find_rel_test():
     install_requires=[
         'setuptools >= 0.6',
         "vsc-base >= 2.4.18",
+        'daemonize >= 2.4.6',
```

Comment:
(Kenneth Hoste, Fri, Mar 25, 2016 at 08:46:59AM -0700:)
> > @@ -113,6 +113,7 @@ def find_rel_test():
> >      install_requires=[
> >          'setuptools >= 0.6',
> >          "vsc-base >= 2.4.18",
> > +        'daemonize >= 2.4.6',
>
> hmm, should we include this as an optional dependency instead (see `extras_require`)?

That's your call as release manager :-) to assess whether this would be
a generally-useful feature that many people could profit from, or a
niche functionality (those who want it will have to go the extra mile).

That said, the `daemonize` Python library is a single file of ~200 lines
with no additional dependencies so I do not see any problem in depending
on it even for the default EB.